src/index.ts

```diff
@@ -1,179 +1,3 @@
-      const normalizedEvent = normalizeIncomingEvent(event);
-
-      const { body, headers, status } = await server.executeHTTPGraphQLRequest({
-        httpGraphQLRequest: normalizedEvent,
-        context: () => contextFunction({ event, context }),
-      });
-
-      if (body.kind === 'chunked') {
-        throw Error('Incremental delivery not implemented');
-      }
-
-      return {
-        statusCode: status || 200,
-        headers: {
-          ...Object.fromEntries(headers),
-          'content-length': Buffer.byteLength(body.string).toString(),
-        },
-        body: body.string,
-      };
-    } catch (e) {
-      return {
-        statusCode: 400,
-        body: (e as Error).message,
-      };
-    }
-  };
-}
-
-function normalizeIncomingEvent(event: IncomingEvent): HTTPGraphQLRequest {
-  let httpMethod: string;
-  if ('httpMethod' in event) {
-    httpMethod = event.httpMethod;
-  } else {
-    httpMethod = event.requestContext.http.method;
-  }
-  const headers = normalizeHeaders(event.headers);
-  let search: string;
-  if ('rawQueryString' in event) {
-    search = event.rawQueryString;
-  } else if ('queryStringParameters' in event) {
-    search = normalizeQueryStringParams(
-      event.queryStringParameters,
-      event.multiValueQueryStringParameters,
-    ).toString();
-  } else {
-    throw new Error('Search params not parsable from event');
-  }
-
-  const body = event.body ?? '';
-
-  return {
-    method: httpMethod,
-    headers,
-    search,
-    body: parseBody(body, headers.get('content-type'), event.isBase64Encoded),
-  };
-}
-
-function parseBody(
-  body: string | null | undefined,
-  contentType: string | undefined,
-  isBase64: boolean,
-): object | string {
-  if (body) {
-    const parsedBody = isBase64
-      ? Buffer.from(body, 'base64').toString('utf8')
-      : body;
-    if (contentType?.startsWith('application/json')) {
-      return JSON.parse(parsedBody);
-    }
-    if (contentType?.startsWith('text/plain')) {
-      return parsedBody;
-    }
-  }
-  return '';
-}
-
-function normalizeHeaders(headers: IncomingEvent['headers']): HeaderMap {
-  const headerMap = new HeaderMap();
-  for (const [key, value] of Object.entries(headers ?? {})) {
-    headerMap.set(key, value ?? '');
-  }
-  return headerMap;
-}
-
-function normalizeQueryStringParams(
-  queryStringParams: Record<string, string | undefined> | null | undefined,
-  multiValueQueryStringParameters:
-    | Record<string, string[] | undefined>
-    | null
-    | undefined,
-): URLSearchParams {
-  const params = new URLSearchParams();
-  for (const [key, value] of Object.entries(queryStringParams ?? {})) {
-    params.append(key, value ?? '');
-  }
-  for (const [key, value] of Object.entries(
-    multiValueQueryStringParameters ?? {},
-  )) {
-    for (const v of value ?? []) {
-      params.append(key, v);
-    }
-  }
-  return params;
-}
+export * from './lambdaHandler';
+export * as handlers from './request-handlers/_index';
```

Comment:
Not opposed to this but have a slight preference toward explicitly and individually exporting each thing.